pkgs/tools/misc/ollama/default.nix

```diff
@@ -165,7 +165,8 @@ goBuild ((lib.optionalAttrs enableRocm {
     # expose runtime libraries necessary to use the gpu
     mv "$out/bin/ollama" "$out/bin/.ollama-unwrapped"
     makeWrapper "$out/bin/.ollama-unwrapped" "$out/bin/ollama" \
-      --suffix LD_LIBRARY_PATH : '/run/opengl-driver/lib:${lib.makeLibraryPath runtimeLibs}'
+      --suffix LD_LIBRARY_PATH : '/run/opengl-driver/lib:${lib.makeLibraryPath runtimeLibs}' '' + lib.optionalString enableRocm ''\
+      --set-default HIP_PATH ${pkgs.rocmPackages.meta.rocm-hip-libraries}
```

Comment:
I think I tested this but without success. If you're not in a hurry I'll try it on Monday and merge if it works.

Reply:
It worked right (used rocm) when I built it. It's not that important though, since it works as is. I'm fine merging this sooner to get this fix into nixpkgs-unstable/nixos-unstable as soon as possible.